src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -265,10 +265,27 @@ combined image rather than an image from each coil.
+| DelayTime                         | RECOMMENDED. User specified time (in seconds) to delay the acquisition of data for the following volume. If the field is not present it is assumed to be set to zero. Corresponds to Siemens CSA header field `lDelayTimeInTR`. This field is REQUIRED for sparse sequences using the `RepetitionTime` field that do not have the `SliceTiming` field set to allowed for accurate calculation of "acquisition time". This field is mutually exclusive with `VolumeTiming`.                                                                                                                               |
 | AcquisitionDuration               | RECOMMENDED. Duration (in seconds) of volume acquisition. Corresponds to DICOM Tag 0018,9073 `Acquisition Duration`. This field is REQUIRED for sequences that are described with the `VolumeTiming` field and that not have the `SliceTiming` field set to allowed for accurate calculation of "acquisition time". This field is mutually exclusive with `RepetitionTime`.                                                                                                                                                                                                                              |
 | DelayAfterTrigger                 | RECOMMENDED. Duration (in seconds) from trigger delivery to scan onset. This delay is commonly caused by adjustments and loading times. This specification is entirely independent of `NumberOfVolumesDiscardedByScanner` or `NumberOfVolumesDiscardedByUser`, as the delay precedes the acquisition.                                                                                                                                                                                                                                                                                                    |
 
+The following table recapitulates the different ways that specific fields have 
+to be populated for functional sequences.
+
+|                   | RepetitionTime | SliceTiming | AcquisitionDuration | DelayTime | VolumeTiming |
+|-------------------|:--------------:|:-----------:|:-------------------:|:---------:|:------------:|
+| option 1          |       \[X]     |             |         \[ ]        |           |      \[ ]    |
+| option 2          |       \[ ]     |     \[X]    |                     |    \[ ]   |      \[X]    |
+| option 3          |       \[ ]     |             |         \[X]        |    \[ ]   |      \[X]    |
+| sparse sequence 1 |       \[X]     |     \[X]    |         \[ ]        |           |      \[ ]    |
+| sparse sequence 1 |       \[X]     |     \[X]    |         \[ ]        |    \[X]   |      \[ ]    |
+| sparse sequence 1 |       \[X]     |             |         \[ ]        |    \[X]   |      \[ ]    |
+
+**Legend**
+\[X] --> has to be filled
+\[ ] --> has to be left empty
+empty cell --> can be specified but not required
```

Comment:
Single new lines are not rendered as new lines in Markdown. Better turn this into a list.